flake8_trio.py

```diff
@@ -1183,6 +1251,21 @@ def add_options(option_manager: OptionManager):
                 "mydecorator,mypackage.mydecorators.*``"
             ),
         )
+        option_manager.add_option(
+            "--startable-in-context-manager",
+            default="",
+            parse_from_config=True,
+            required=False,
+            comma_separated_list=True,
+            help=(
+                "Comma-separated list of method calls to enable TRIO113"
+                " warnings for."
+                " Use if you want to override the default methods the check is enabled for."

```

Comment:
Very fair that you didn't bother with two options - but this is no longer true, it extends the defaults.